cpp/include/cudf/types.hpp

```diff
@@ -342,5 +342,6 @@ enum class hash_id {
  */
 static constexpr uint32_t DEFAULT_HASH_SEED = 0;
 
+typedef __int128_t int128;
```

Comment:
is this really needed? Seems a bit risky.

Reply:
This is a bit of a hack we had to do for cython to support `int128`: https://stackoverflow.com/a/27583541

Reply:
cc: @codereport for visibility